generated/java/proto-google-cloud-bigquerydatatransfer-v1/src/main/java/com/google/cloud/bigquery/datatransfer/v1/ListTransferRunsResponse.java

```diff
@@ -34,6 +34,9 @@ private ListTransferRunsResponse(
       com.google.protobuf.ExtensionRegistryLite extensionRegistry)
       throws com.google.protobuf.InvalidProtocolBufferException {
     this();
+    if (extensionRegistry == null) {
```

Comment:
@garrettjonesgoogle Is this proto file change (which we see in all the proto classes) expected?

Reply:
I don't recognize this change - It doesn't seem like a reversion

Reply:
I believe it is caused by this update to protoc in artman: https://github.com/googleapis/artman/pull/401/files#diff-3254677a7917c6c01f55212f86c57fbfR36